guides/common/modules/proc_listing-using-satellite-ansible-modules.adoc

```diff
@@ -0,0 +1,49 @@
+[id="listing-using-satellite-ansible-modules_{context}"]
+== Listing the {Project} Ansible Modules
```

Comment:
I suggest renaming this to *Viewing the {Project} Ansible Modules*. Listing sounds very narrow. What do you think @suyogsainkar ?

Reply:
Good suggestion, done.